programs/nft-voter/src/instructions/create_voter_weight_record.rs

```diff
@@ -0,0 +1,64 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::Mint;
+use spl_governance::state::realm;
+
+use crate::state::VoterWeightRecord;
+
+/// Creates VoterWeightRecord used by spl-gov
+/// This instruction should only be executed once per realm/governing_token_mint/governing_token_owner
+/// to create the account
+#[derive(Accounts)]
+#[instruction(governing_token_owner: Pubkey)]
```

Comment:
nitpick: so the owner doesnt need to be the signer? is this intended

Reply:
Correct, this is one time account allocation action without any adverse effects and hence can be executed by anybody who wants to pay the rent